routes/v1.js

```diff
@@ -1236,10 +1271,11 @@ api.declare({
   name:       'reportCompleted',
   scopes: [
     [
+      // Legacy
       'queue:resolve-task',
       'assume:worker-id:<workerGroup>/<workerId>'
     ], [
-      'queue:claim-task:<taskId>/<runId>'
+      'queue:claim-run-id:<taskId>/<runId>'
```

Comment:
I think we should optimize for the common flow... where workers have  `queue:claim-run-id:<taskId>/<runId>` from the temp creds issued by claimTask... And not complicate it... we can always add another flow/scope options...